src/cc/libclient/KfsClient.cc

```diff
@@ -6839,13 +6890,16 @@ KfsClientImpl::CompareChunkReplicas(const char* pathname, string& md5sum)
 int
 KfsClientImpl::GetChunkLease(
     kfsChunkId_t       inChunkId,
+    int64_t            inChunkVersion,
     const char*        inPathNamePtr,
     int                inLeaseTime,
     ChunkServerAccess& inChunkServerAccess,
     int64_t&           outLeaseId)
 {
     LeaseAcquireOp theLeaseOp(0, inChunkId, inPathNamePtr);
     theLeaseOp.leaseTimeout = inLeaseTime;
+    theLeaseOp.chunkPos     = inChunkVersion < 0 ?
```

Comment:
It seems we are doing this operation a lot. Is this a common operation. Should it be placed into a library function and given a more semantic name?